Makefile

```diff
@@ -134,6 +138,10 @@ $(LIBRLN_FILE):
 
 librln: | $(LIBRLN_FILE)
 	$(eval NIM_PARAMS += --passL:$(LIBRLN_FILE) --passL:-lm)
+ifeq ($(RLN_V2),true)
```

Comment:
I have the impression that we won't test this in our CI. Shall we switch it manually in the future? To elaborate a bit more, I think we'd need to adapt the following somehow in the future: https://github.com/waku-org/nwaku/blob/bb58a63aa16ebd50ee243e17b58b81500da554e1/.github/workflows/ci.yml#L121

Reply:
yes, at the moment rln-v2 is unstable in nwaku. when it's at a better state I'll add the flag similar to how we had it for experimental builds